BootloaderCorePkg/BootloaderCorePkg.dsc

```diff
@@ -247,6 +247,24 @@
   gPlatformCommonLibTokenSpaceGuid.PcdUsbKeyboardPollingTimeout | $(USB_KB_POLLING_TIMEOUT)
   gPlatformCommonLibTokenSpaceGuid.PcdLowestSupportedFwVer		| $(LOWEST_SUPPORTED_FW_VER)
 
+  !if $(RSA_SIGN_TYPE) == RSA2048
+    gPlatformCommonLibTokenSpaceGuid.PcdRsaSignType | 0x1
+  !elseif $(RSA_SIGN_TYPE) == RSA3072
+    gPlatformCommonLibTokenSpaceGuid.PcdRsaSignType | 0x2
+  !endif
+
+  !if $(SIGN_HASH_TYPE) == SHA2_256
+    gPlatformCommonLibTokenSpaceGuid.PcdSignHashType | 0x0
+  !elseif $(SIGN_HASH_TYPE) == SHA2_384
+    gPlatformCommonLibTokenSpaceGuid.PcdSignHashType | 0x1
+  !elseif $(SIGN_HASH_TYPE) == SHA2_512
+    gPlatformCommonLibTokenSpaceGuid.PcdSignHashType | 0x2
+  !elseif $(SIGN_HASH_TYPE) == SM3_256
+    gPlatformCommonLibTokenSpaceGuid.PcdSignHashType | 0x3
+  !endif
+
+  gPlatformCommonLibTokenSpaceGuid.PcdIppHashLibSupported    | 0xB
```

Comment:
Need to understand what 0xB means.

Reply:
Definition for PcdIpplibSupported 
 0x0001    - SHA2_256.
0x0002    - SHA2_384.
 0x0004    - SHA2_512.
0x0008    - SM3_256.

We currently have libs included - SHA3256, SHA384, SM3 only. Based on this setting lib calls would be included.